.yarn/patches/@tanstack-react-router-npm-1.97.3-ad16343f36.patch

```diff
@@ -0,0 +1,192 @@
+diff --git a/dist/cjs/router.cjs b/dist/cjs/router.cjs
```

Comment:
See 
https://github.com/TanStack/router/pull/2698
and
https://github.com/TanStack/router/pull/3161